pointmatcher/ErrorMinimizersImpl.h

```diff
@@ -95,30 +95,6 @@ struct ErrorMinimizersImpl
 		virtual T getOverlap() const;
 	};
 
-	struct PointToPlaneErrorMinimizer: public ErrorMinimizer
```

Comment:
One of the idea was to allow user to build an ICP solution step by step in their code using the provided building blocks. I doubt that it is used much, but I think it's still a pertinent feature.